src/communication_managers/iothub_communication_manager/inc/aduc/iothub_communication_manager.h

```diff
@@ -64,11 +64,11 @@ typedef void (*ADUC_COMMUNICATION_MANAGER_CLIENT_HANDLE_UPDATED_CALLBACK)(ADUC_C
  *
  * @return 'true' if success.
  */
-bool IoTHub_CommunicationManager_Init(
-    ADUC_ClientHandle* handle_address,
-    IOTHUB_CLIENT_DEVICE_TWIN_CALLBACK device_twin_callback,
-    ADUC_COMMUNICATION_MANAGER_CLIENT_HANDLE_UPDATED_CALLBACK client_handle_updated_callback,
-    ADUC_PnPComponentClient_PropertyUpdate_Context* property_update_context);
+bool IoTHub_CommunicationManager_Init(ADUC_ClientHandle* handle_address,
```

Comment:
nit: This is the first formatting weirdness I've seen unless it's just my viewer.

Reply:
Those formatted lines were introduced by [this commit in develop](https://github.com/Azure/iot-hub-device-update/commit/0644bdfbe1487c5ad8458f05ee8f82946cb2eb4f) so it probably auto-merged this way. I'm assuming it's from clang-format or maybe an IDE extension, etc. Need to get the clang-format git hook working one of these days ...